pkgs/by-name/ru/rust-streamdeck/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  lib,
+  udev,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "rust-streamdeck";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "ryankurte";
+    repo = "rust-streamdeck";
+    rev = "v${version}";
+    hash = "sha256-9FuTnRQHKYJzMqhhgyTVq2R+drn4HAr3GDNjQgc3r+w=";
+  };
+
+  postPatch = ''
```

Comment:
there is a vendoring primitive for cargo

there is a lot of examples

Reply:
I'm not sure exactly what to look for, looking for `Cargo.lock` gave examples identical to mine (i.e. like in the doc).